test/mx/distribution/test_mixture.py

```diff
@@ -52,12 +51,12 @@ def plot_samples(s: Tensor, bins: int = 100) -> None:
 BINS = np.linspace(-5, 5, 100)
 
 
-def histogram(samples: NPArrayLike) -> np.ndarray:
+def histogram(samples: np.ndarray) -> np.ndarray:
     h, _ = np.histogram(samples, bins=BINS, density=True)
     return h
 
 
-def diff(x: NPArrayLike, y: NPArrayLike) -> np.ndarray:
+def diff(x: np.ndarray, y: np.ndarray) -> np.ndarray:
```

Comment:
This is also a terrible name: it’s the mean absolute error being computed here

Reply:
One fix at a time!